Rakefile

```diff
@@ -1,12 +1,22 @@
+# frozen_string_literal: true
+
 require 'bundler'
 Bundler::GemHelper.install_tasks
 
-require 'rspec/core/rake_task'
-require 'spree/testing_support/extension_rake'
+begin
+  require 'spree/testing_support/extension_rake'
+  require 'rubocop/rake_task'
+  require 'rspec/core/rake_task'
+  RSpec::Core::RakeTask.new(:spec)
+
+  RuboCop::RakeTask.new
 
-RSpec::Core::RakeTask.new
+  task default: %i(first_run rubocop spec)
+rescue LoadError # rubocop:disable Lint/HandleExceptions
+  # no rspec available
+end
 
-task :default do
+task :first_run do # rubocop:disable Rails/RakeEnvironment
```

Comment:
with this change, will we need to invoke `bundle exec rake first_run` to prepare the dummy app and everything else that's required?

Reply:
No need, the new `default` task includes `first_run`.